test/pt2_to_circle_test/test_pt2_to_circle.py

```diff
@@ -227,14 +227,23 @@ def validate_result(
                 desired=circle_res.shape,
                 err_msg=f"Shape mismatches.\nexpected result: {expected_res.shape}\ncircle result: {circle_res.shape}",
             )
-        np.testing.assert_allclose(
-            actual=circle_res,
-            desired=expected_res,
-            rtol=rtol,
-            atol=atol,
-            err_msg=f"Value mismatches.\nexpected result: {expected_res}\ncircle result: {circle_res}",
-        )
-
+            circle_tensor = torch.from_numpy(circle_res)
+            torch.testing.assert_close(
+                actual=circle_tensor,
+                expected=expected_res,
+                equal_nan=True,
```

Comment:
> NaN’s are only considered equal to each other if equal_nan is True.
From https://docs.pytorch.org/docs/stable/testing.html

I found that some unittests return `NaN` value.